.github/workflows/run_g_tests.yml

```diff
@@ -6,15 +6,39 @@ on:
 
 jobs:
   run_g_tests:
-    name: Run G Tests
-    runs-on: self-hosted
+    name: Run G Tests LV${{ matrix.version }}
+    runs-on: 
+      - self-hosted
+      - windows
+      - x64
+      - rdss-measlinkbot-win-10-lv64
+    strategy:
+      matrix:
+        version: ["2020", "2021", "2022", "2023"]
```

Comment:
All these going to run in parallel? Right now our PR build is about 6 minutes. I hope it doesn't go up to 24 minutes.

Reply:
The matrix feature maximizes parallelism by default.
https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#defining-the-maximum-number-of-concurrent-jobs

Reply:
Implemented this suggestion in PR [here](https://github.com/ni/measurementlink-labview/pull/509).